docs/ops/security-ssl.md

```diff
@@ -33,6 +33,9 @@ SSL can be enabled for all network communication between Flink components. SSL k
 * **akka.ssl.enabled**: SSL flag for akka based control connection between the Flink client, jobmanager and taskmanager 
 * **jobmanager.web.ssl.enabled**: Flag to enable https access to the jobmanager's web frontend
 
+Please see the configuration page about the
+[complete list of SSL configuration parameters]({{site.baseurl}}/ops/config.html#ssl-settings), in particular **security.ssl.session-cache-size**.
```

Comment:
just a suggestion, you could also embed the entire table directly, see `Configuration.md` on how to do it.

Reply:
agreed, that would make sense